bedrock/firefox/views.py

```diff
@@ -579,14 +579,11 @@ def get_template_names(self):
             else:
                 template = "firefox/whatsnew/index.html"
         else:
-            if show_default_account_whatsnew(version) and ftl_file_is_active("firefox/whatsnew/whatsnew-account"):
```

Comment:
**Suggestion:** if we not longer need it, let's remove the `show_default_account_whatsnew()` function?